src/mono/mono/mini/interp/transform.c

```diff
@@ -359,17 +359,19 @@ interp_make_var_renamable (TransformData *td, int var)
 	if (td->vars [var].ext_index != -1)
 		return td->vars [var].ext_index;
 
-	if (td->renamable_vars_size == td->renamable_vars_capacity) {
-		td->renamable_vars_capacity *= 2;
-		if (td->renamable_vars_capacity == 0)
-			td->renamable_vars_capacity = 2;
-		td->renamable_vars = (InterpRenamableVar*) g_realloc (td->renamable_vars, td->renamable_vars_capacity * sizeof (InterpRenamableVar));
-	}
-
 	int ext_index = td->renamable_vars_size;
-	InterpRenamableVar *ext = &td->renamable_vars [ext_index];
-	memset (ext, 0, sizeof (InterpRenamableVar));
-	ext->var_index = var;
+
```

Comment:
`interp_make_var_renamable` method can only be called in optimized code. Don't understand what these changes are supposed to do.

Reply:
OK, wasn't sure if ext_index could get touched outside of the optimizer. I'll clean it up, thanks